scalyr_agent/agent_main.py

```diff
@@ -249,7 +249,7 @@ def main(self, config_file_path, command, command_options):
                 self.__config = None
                 print(
                     "Could not parse configuration file at '%s'" % config_file_path,
-                    file=sys.stderr
+                    file=sys.stderr,
```

Comment:
Why is there a comma at the end of this line?  This is repeated many times below.  Can we remove the comma and in the other places as well?